spark-images/spark3.Dockerfile

```diff
@@ -89,11 +89,8 @@ ENV LD_LIBRARY_PATH /lib64
 
 WORKDIR /opt/spark/work-dir
 RUN chmod g+w /opt/spark/work-dir
-# RUN chmod a+x /opt/decom.sh 
+# RUN chmod a+x /opt/decom.sh
 
-RUN mkdir -p /etc/metrics/conf
```

Comment:
Why do you remove the metrics configuration?

Reply:
Yes, I removed it for my own deployment since I don't know how this work and I don't have prometheus, but it is a mistake on my side to have that update in the pull request. I put them back.